base/stream.jl

```diff
@@ -1193,9 +1193,9 @@ unmark(x::LibuvStream)   = unmark(x.buffer)
 reset(x::LibuvStream)    = reset(x.buffer)
 ismarked(x::LibuvStream) = ismarked(x.buffer)
 
-function peek(s::LibuvStream)
+function peek(s::LibuvStream, ::Type{T}) where T
```

Comment:
Seems to be identical to the fallback method. Not a problem though.

Reply:
Yeah, I noticed that. Figured I'd punt on dealing with that though, just in case there was something I was missing.